packages/core/src/components/ColorPalette/ColorPicker.tsx

```diff
@@ -57,6 +57,15 @@ function DropdownInput({
     setHexInputValue(tinycolor(colorPickerColor).toHexString())
   }, [colorPickerColor])
 
+  const updatePrimaryColorToValidColor = () => {
```

Comment:
@noobCode-69 Do you think also calling this function while unmounting this component will help to solve the problem?

Reply:
Yeah , It could solve it but there are some problems
- The ColorPicker.tsx is having it's local state to store the input
- The index.tsx of the onboarding also have a state that store the input
- The ColorPicker.tsx is updating the state of index.tsx with it's local state when the input get blurred
The solution that you suggest won't work because of the third point.